### PR TITLE
enable all tests to run on LRE env

### PR DESCRIPTION
**Pull request by bhavaniravichandran**
Fixed logging and metrics tests to run in unique namespaces and added them to lre Jenkinsfile

### CODE REVIEW

File: ci/lre/Jenkinsfile [hunk 1]
```diff
@@ -1,12 +1,12 @@
 // Copyright (c) 2022, Oracle and/or its affiliates.
 // Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
 
-properties([[$class: 'ThrottleJobProperty', categories: [], limitOneJobWithMatchingParams: false, maxConcurrentPerNode: 1,
-                    maxConcurrentTotal: 1, paramsToUseForLimit: '', throttleEnabled: true, throttleOption: 'project'],
-               [$class: 'BuildDiscarderProperty',
-                strategy: [$class: 'LogRotator', numToKeepStr: '10']],
-                pipelineTriggers([cron('H/5 * * * *')]),
-           ])
+//properties([[$class: 'ThrottleJobProperty', categories: [], limitOneJobWithMatchingParams: false, maxConcurrentPerNode: 1,
```
Comment (markxnelson) [inline ci/lre/Jenkinsfile:4]: just delete it

File: ci/lre/Jenkinsfile [hunk 2]
```diff
@@ -333,7 +333,26 @@ pipeline {
                         runGinkgo('metrics/deploymetrics')
                     }
                 }
-
+                stage('poko metricsbinding') {
+                    steps {
+                        runGinkgo('metricsbinding')
```
Comment (markxnelson) [inline ci/lre/Jenkinsfile:338]: can we consolidate all of this into one big runGinkgo('a', 'b', 'c', 'd' ... ) ?

File: tests/e2e/metricsbinding/testdata/hello-helidon-replicaset.yaml [hunk 3]
```diff
@@ -5,7 +5,6 @@ apiVersion: apps/v1
 kind: ReplicaSet
 metadata:
   name: hello-helidon-replicaset
```
Comment (markxnelson) [inline tests/e2e/metricsbinding/testdata/hello-helidon-replicaset.yaml:7]: make sure the corresponding docs updates have been made in master and release-1.2 branches in the docs repo too please

**Review by markxnelson — APPROVED** [review]
a couple of comments - you can fix them in another pr after this one

**Review by stevshap — APPROVED** [review]

**Pull request merged**